test/screenshot/spec/mdc-textfield/fixture.js

```diff
@@ -22,7 +22,7 @@
  */
 
 window.mdc.testFixture.fontsLoaded.then(() => {
-  [].forEach.call(document.querySelectorAll('.mdc-text-field'), (el) => {
+  [].forEach.call(document.querySelectorAll('.mdc-text-field:not([data-no-init="true"])'), (el) => {
```

Comment:
Do we have anywhere in our docs that you shouldn't initialize a disabled text field?

Reply:
Internal implementation does not initialize the component until interacted. In that case asterisk would still be in red color.

Reply:
I'm suggesting that we need to add some text to the Readme so that developers know not to initialize the text field until disabled is removed.

Reply:
Clients can definitely initialize text field even if it is disabled, why do you think they don't need to initialize when it is disabled?

Reply:
I'm just wondering why you aren't initializing the text field.

Reply:
I thought you said this morning that initializing it will remove `--invalid` class.

Reply:
Two reasons for this screenshot without initializing text field:

- Validates the styles (Only style without JS) for disabled & invalid state. Our screenshots are captured after JS execution that removes `--invalid` modifier from root. There might be some uses cases where clients do not need to initialize text field for some reason.
- Our internal implementation, we do not instantiate the foundation until the text field is interacted hence the asterisk stays red on page load which is unexpected. This test case should cover this case.